examples/vision/siamese.py

```diff
@@ -0,0 +1,345 @@
+
+"""
+# Siamese network with a contrastive loss
+
+Author: Mehdi<br>
+Date created: 2021/05/06<br>
+Last modified: 2020/05/06<br>
+Description: Similarity learning using siamese network with contrastive loss
```

Comment:
Please fix the header -- see other code examples `.py` files for what the header should look like